modules/twinklearv.js

```diff
@@ -80,6 +80,21 @@ Twinkle.arv.callback = function (uid, isIP) {
 		id: 'twinkle-arv-blockwarning'
 	});
 
+	// Temporary account notice
+	if (mw.config.get('wgRelevantUserName') && mw.util.isTemporaryUser(mw.config.get('wgRelevantUserName'))) {
+		const temporaryAccountNotice = form.append({
+			type: 'field',
+			label: 'Temporary account notice',
+			name: 'ta_notice',
+			style: 'color: var(--morebits-color-warning, #FF4500)'
+		});
+
+		temporaryAccountNotice.append({
+			type: 'div',
+			label: 'You are reporting a new temporary account. These accounts, with usernames like ~2025-12345-67, have replaced IP addresses. Before reporting any temporary accounts, please familiarize yourself with [[Wikipedia:Temporary accounts|how they work]].'
```

Comment:
Sorry, one more comment. This message is kind of vague. Is there a specific incident or common misunderstanding that was the motivation for adding this? Like a certain kind of incorrect AIV report? Maybe it'd be better to give specific advice about whatever that is, rather than just linking to [[WP:TA]].